code/components/jomjol_wlan/read_wlanini.cpp

```diff
@@ -119,6 +120,19 @@ void LoadWlanFromFile(std::string fn, char *&_ssid, char *&_password, char *&_ho
             }
         }
 
+        if ((zerlegt.size() > 1) && (toUpper(zerlegt[0]) == "HTTP_USERNAME")){
+            http_username = zerlegt[1];
+            if ((http_username[0] == '"') && (http_username[http_username.length()-1] == '"')){
```

Comment:
`std::string` supports `front()` and `back()` to access the first and last character. So you could do this to make it more readable and also safer (no "magic numbers"):
```suggestion
            if (http_username.front() == '"' && http_username.back() == '"')
            {
```